appveyor.yml

```diff
@@ -7,7 +7,7 @@ skip_branch_with_pr: true
 build:
   verbosity: minimal
 
-test: off
+test: false

```

Comment:
Yaml boolean preference or did this fix something?

Reply:
My Visual Studio Code YAML linter complains about `off` as a boolean value.